cfgov/v1/util/util.py

```diff
@@ -143,3 +144,14 @@ def all_valid_destinations_for_request(request):
                             valid_destination_for_request(request, pair[1])]
 
     return valid_destinations
+
+
+def get_streamfields(page):
+    """
+    Retrieves the stream values on a page from it's Streamfield
```

Comment:
This method actually uses the name of the Streamfield on the page as a key in the dictionary, so this wouldn't happen. The dictionary for this example page would be:
```python
{
    'section1': [block, block, block, ...],
    'section2': [block, block, block, ...]
}
```